chat/input-bar/src/InputBar/InputBar.tsx

```diff
@@ -382,6 +387,21 @@ export const InputBar = forwardRef<HTMLFormElement, InputBarProps>(
                 ref={textareaRef}
               />
               <div className={rightContentStyles}>
+                {shouldRenderHotkeyIndicator && !disabled && (
+                  <div
+                    data-testid="hotkey-indicator"
+                    className={cx(
+                      baseHotkeyIndicatorStyles,
+                      themedHotkeyIndicatorStyles[theme],
+                      {
+                        [hotkeyIndicatorFocusedStyles]: isFocused,
+                        [hotkeyIndicatorUnfocusedStyles]: !isFocused,
+                      },
+                    )}
+                  >
+                    /
+                  </div>
```

Comment:
As far as I know, this is just a visual indicator that hitting slash opens up the chatbot. I suppose the intended use case is for the user to open up the chatbot via the button initially, see the hotkey indicator, and then subsequently use the hotkey to open the chatbot.

If you have any suggestions on additional accessibility attributes or extra context via comments or otherwise, I'm happy to add them!

Reply:
Is the functionality being baked in on the consuming application side? Was the point of this prop to be strictly presentational? I feel like if we're adding the prop, maybe we want to couple it with the functionality that opens the window when the `/` key is hit

Reply:
Yeah the extent of it in leafygreen was to just be presentational. Correct me if I'm wrong as I'm quite unfamiliar with both codebases, but I was under the impression that handling opening/closing of the window was handled by the [chatbot](https://github.com/mongodb/chatbot/blob/main/packages/mongodb-chatbot-ui/) `mongodb-chatbot-ui` package. My plan was to couple the functionality in that repository.

Reply:
OK sounds good.